README.md

```diff
@@ -589,9 +589,9 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <tr>
       <td align="center" valign="top" width="14.28%"><a href="https://juhoholsa.com/"><img src="https://avatars.githubusercontent.com/u/15036353?v=4" loading="lazy" width="100" alt=""/><br /><sub><b>Juho Hölsä</b></sub></a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/seifer7"><img src="https://avatars.githubusercontent.com/u/13601073?v=4" loading="lazy" width="100" alt=""/><br /><sub><b>Kane</b></sub></a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Sdfendor"><img src="https://avatars.githubusercontent.com/u/2728018?v=4" loading="lazy" width="100" alt=""/><br /><sub><b>Kevin Jakob</b></sub></a></td>
```

Comment:
I do not know why this keeps getting deleted every time there is a new contributor.

Reply:
Thanks, Ng Kiat Siong, I (obviously) hadn't noticed yarn removed those lines when adding my profile.

Reply:
+1 for https://github.com/OpenMage/magento-lts/issues/2880 again (?)

Reply:
yes, that should be automated or anyway not touched manually

Reply:
@fballiano https://allcontributors.org/docs/en/bot/installation